src/cbfs.c

```diff
@@ -0,0 +1,218 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#include "cbfs.h"
+
+#include <unistd.h>
+
+#include <errno.h>
+#include <stdbool.h>
+#include <stdio.h>
+
+#include "boot_data.h"
+#include "utils.h"
+
+#define BOOTORDER_REGION "BOOTORDER"
```

Comment:
the bootorder region is available since a few months. Previously we had kept the runtime configuration in the bootorder BCFS raw file like the def and map files. The bootorder file was also 4K aligned with 4K size (padded). @SergiiDmytruk we should enable support for this mechanism as well, since legacy firmware (4.0.x) is still using the bootorder CBFS file. The logics should look like this:
If (bootorder region is not found) {
    use bootorder CBFS file
} else {
    use bootorder region
}